File: backend/backend-analyzer/backend-analyzer/Models/Session.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace AnalyzerServer.Models
{
    public class Session
    {
        public int Id { get; set; }

        [Required]
        public int UserId { get; set; }

        [Required]
        public string SessionToken { get; set; }

        [Required]
        public DateTime ExpiresAt { get; set; }
    }
}

File: backend/models/session.js
module.exports = (sequelize, DataTypes) => {
    return sequelize.define('Session', {
        user_id: { type: DataTypes.INTEGER, allowNull: false },
        session_token: { type: DataTypes.TEXT, unique: true, allowNull: false },
        expires_at: { type: DataTypes.DATE, allowNull: false },
    }, { tableName: 'sessions', timestamps: false });
};
